{
  "uid" : "a75ffed81ebd16fe",
  "name" : "NHS Number: Searching with invalid NHS number shows an error [9753108642-Enter a correct NHS number]",
  "fullName" : "/app/features/find_a_patient.feature:NHS Number: Searching with invalid NHS number shows an error",
  "historyId" : "4cb2e2253d26b616069dd8a9fc22175b",
  "time" : {
    "start" : 1730188467909,
    "stop" : 1730188611081,
    "duration" : 143172
  },
  "status" : "failed",
  "statusMessage" : "AssertionError: assert False is True\n +  where False = check_nhs_number_error_message_text_exists('Enter a correct NHS number')",
  "statusTrace" : "fixturefunc = <function step_error_message_appears_for_nhs_number at 0x7f0f3a655620>\nrequest = <FixtureRequest for <Function test_nhs_number_searching_with_invalid_nhs_number_shows_an_error[9753108642-Enter a correct NHS number]>>\nkwargs = {'errorMessage': 'Enter a correct NHS number'}\n\n    def call_fixture_func(\n        fixturefunc: _FixtureFunc[FixtureValue], request: FixtureRequest, kwargs\n    ) -> FixtureValue:\n        if is_generator(fixturefunc):\n            fixturefunc = cast(\n                Callable[..., Generator[FixtureValue, None, None]], fixturefunc\n            )\n            generator = fixturefunc(**kwargs)\n            try:\n                fixture_result = next(generator)\n            except StopIteration:\n                raise ValueError(f\"{request.fixturename} did not yield a value\") from None\n            finalizer = functools.partial(_teardown_yield_fixture, fixturefunc, generator)\n            request.addfinalizer(finalizer)\n        else:\n            fixturefunc = cast(Callable[..., FixtureValue], fixturefunc)\n>           fixture_result = fixturefunc(**kwargs)\n\n.tox/py311/lib/python3.11/site-packages/_pytest/fixtures.py:898: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nerrorMessage = 'Enter a correct NHS number'\n\n    @then(parse('I can see an nhs number error message {errorMessage}'))\n    def step_error_message_appears_for_nhs_number(errorMessage):\n        attach_screenshot(\"error_message_appears_for_nhs_number\")\n>       assert check_nhs_number_error_message_text_exists(errorMessage) is True\nE       AssertionError: assert False is True\nE        +  where False = check_nhs_number_error_message_text_exists('Enter a correct NHS number')\n\ntests/test_find_a_patient_steps.py:135: AssertionError",
  "flaky" : false,
  "newFailed" : false,
  "newBroken" : false,
  "newPassed" : false,
  "retriesCount" : 0,
  "retriesStatusChange" : false,
  "beforeStages" : [ ],
  "testStage" : {
    "status" : "failed",
    "statusMessage" : "AssertionError: assert False is True\n +  where False = check_nhs_number_error_message_text_exists('Enter a correct NHS number')",
    "statusTrace" : "fixturefunc = <function step_error_message_appears_for_nhs_number at 0x7f0f3a655620>\nrequest = <FixtureRequest for <Function test_nhs_number_searching_with_invalid_nhs_number_shows_an_error[9753108642-Enter a correct NHS number]>>\nkwargs = {'errorMessage': 'Enter a correct NHS number'}\n\n    def call_fixture_func(\n        fixturefunc: _FixtureFunc[FixtureValue], request: FixtureRequest, kwargs\n    ) -> FixtureValue:\n        if is_generator(fixturefunc):\n            fixturefunc = cast(\n                Callable[..., Generator[FixtureValue, None, None]], fixturefunc\n            )\n            generator = fixturefunc(**kwargs)\n            try:\n                fixture_result = next(generator)\n            except StopIteration:\n                raise ValueError(f\"{request.fixturename} did not yield a value\") from None\n            finalizer = functools.partial(_teardown_yield_fixture, fixturefunc, generator)\n            request.addfinalizer(finalizer)\n        else:\n            fixturefunc = cast(Callable[..., FixtureValue], fixturefunc)\n>           fixture_result = fixturefunc(**kwargs)\n\n.tox/py311/lib/python3.11/site-packages/_pytest/fixtures.py:898: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nerrorMessage = 'Enter a correct NHS number'\n\n    @then(parse('I can see an nhs number error message {errorMessage}'))\n    def step_error_message_appears_for_nhs_number(errorMessage):\n        attach_screenshot(\"error_message_appears_for_nhs_number\")\n>       assert check_nhs_number_error_message_text_exists(errorMessage) is True\nE       AssertionError: assert False is True\nE        +  where False = check_nhs_number_error_message_text_exists('Enter a correct NHS number')\n\ntests/test_find_a_patient_steps.py:135: AssertionError",
    "steps" : [ {
      "name" : "Given I am on the find a patient by nhs number page",
      "time" : {
        "start" : 1730188467910,
        "stop" : 1730188560225,
        "duration" : 92315
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    }, {
      "name" : "And I enter 9753108642 as the nhs number",
      "time" : {
        "start" : 1730188560226,
        "stop" : 1730188580550,
        "duration" : 20324
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    }, {
      "name" : "When I click the search button",
      "time" : {
        "start" : 1730188580551,
        "stop" : 1730188600877,
        "duration" : 20326
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    }, {
      "name" : "Then I can see an nhs number error message Enter a correct NHS number",
      "time" : {
        "start" : 1730188600877,
        "stop" : 1730188611081,
        "duration" : 10204
      },
      "status" : "failed",
      "statusMessage" : "assert False is True\n +  where False = check_nhs_number_error_message_text_exists('Enter a correct NHS number')",
      "statusTrace" : "AssertionError: assert False is True\n +  where False = check_nhs_number_error_message_text_exists('Enter a correct NHS number')\n",
      "steps" : [ ],
      "attachments" : [ {
        "uid" : "c83dd27fcd8dff65",
        "name" : "dev_chrome_130.0.6723.69_error_message_appears_for_nhs_number.png",
        "source" : "c83dd27fcd8dff65.png",
        "type" : "image/png",
        "size" : 4331
      } ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 1,
      "shouldDisplayMessage" : true,
      "hasContent" : true
    } ],
    "attachments" : [ {
      "uid" : "499cfb10bc62feda",
      "name" : "log",
      "source" : "499cfb10bc62feda.txt",
      "type" : "text/plain",
      "size" : 1617
    } ],
    "parameters" : [ ],
    "stepsCount" : 4,
    "attachmentsCount" : 2,
    "shouldDisplayMessage" : true,
    "hasContent" : true
  },
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "host",
    "value" : "7c0be77e3f3d"
  }, {
    "name" : "thread",
    "value" : "1102-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest-bdd"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "feature",
    "value" : "Find a patient"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ {
    "name" : "errorMessage",
    "value" : "Enter a correct NHS number"
  }, {
    "name" : "nhsNumber",
    "value" : "9753108642"
  } ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Product defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "tags" : [ ]
  },
  "source" : "a75ffed81ebd16fe.json",
  "parameterValues" : [ "Enter a correct NHS number", "9753108642" ]
}